src/eventhandler/EventHandler_SceneItems.cpp

```diff
@@ -228,7 +228,7 @@ void EventHandler::HandleSceneItemSelected(void *param, calldata_t *data)
  *
  * @dataField sceneName          | String | The name of the scene the item is in
  * @dataField sceneItemId        | Number | Numeric ID of the scene item
- * @dataField sceneItemTransform | Object | New transform/crop info of the scene item
+ * @dataField sceneItemTransform | Partial<SceneItemTransform> | New transform/crop info of the scene item
```

Comment:
You're introducing the concept of a partial object (which is fine of course), but I don't see this specified anywhere that it's a typedef within an array. Is there something I'm missing?

In theory, pretty much every returned object in obs-websocket is partial. Perhaps it would be better to just make a blanket statement calling all objects returned by obs-websocket "partial"?

Reply:
That particular line is a mistake and was not meant to be a partial. It will always return the full `SceneItemTransform` object every time.

Where `Partial`s are meant to be used is in requests such as `RequestHandler::SetSceneItemTransform`, because the requestor is NOT required to specify a value for every property, only the properties they intend to update. I think this distinction is worth specifying via the `Partial` keyword. However, as it stands now, `Partial` is used in exactly one place.